RELEASE/scripts/autoscend/auto_post_adv.ash

```diff
@@ -323,6 +323,27 @@ boolean auto_post_adventure()
 		}
 		return true;
 	}
+	if(in_aosol())
+	{
+		if(my_class() == $class[Pig Skinner])
+		{
+			buffMaintain($effect[Cheerled], 30, 1, 10);
+			buffMaintain($effect[Taped Up], 20, 1, 10);
+			//buffMaintain($effect[Stretched], 10, 1, 10); In Providers
+		}
+		if(my_class() == $class[Cheese Wizard])
+		{
+			//buffMaintain($effect[Shifted Reality], 25, 1, 10);  In Providers
+			buffMaintain($effect[Cheddarmored], 5, 1, 10);
+			buffMaintain($effect[Queso Fustulento], 10, 1, 10);
```

Comment:
Only 1 mp per turn but I don't see much use for it. The skills the classes get seem to have good elemental damage. Are boss fights hard in this path?

Reply:
Corruptor shadow (Protector Spectre in the Massive Ziggurat), in particular, is very difficult compared to Standard. Damage is capped at 10 for all sources and it is 100% physical resistant. All other bosses are 80-100% physical resistant, except the Goblin King's Shadow, that is 50% physical resistant, according to Mafia

Reply:
Normal combats are pretty trivial because of the big stat buffs we get combined with the stat based damage. Moved Queso Fustulento and Tricky Timpani to all boss fights.